packages/commerce-sdk-react/src/hooks/ShopperOrders/cache.ts

```diff
@@ -5,16 +5,16 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {getCustomerBaskets} from '../ShopperCustomers/queryKeyHelpers'
-import {ApiClients, CacheUpdateMatrix, CacheUpdateUpdate, CacheUpdateInvalidate} from '../types'
+import {ApiClients, Argument, CacheUpdateMatrix, CacheUpdate, CacheUpdateUpdate, CacheUpdateInvalidate, MergedOptions} from '../types'
 import {getOrder} from './queryKeyHelpers'
 
 type Client = ApiClients['shopperOrders']
+/** Parameters that get passed around, includes client config and possible parameters from other endpoints */
+type GetOrderParameters = MergedOptions<Client, Argument<Client['getOrder']>>['parameters']
```

Comment:
```suggestion
type GetOrderOptions = MergedOptions<Client, Argument<Client['getOrder']>>
```
[part 1] save a layer of nesting 😉